### ISSUE TITLE
Reminders don't work on Oreo.

### ISSUE THREAD

**Issue by Forbo** [glesik/wifireminders#16]
Pixel running Android 8.1, reminders don't appear to work at all. Tried multiple reminders, cycling wireless status (off, disconnected, connected), changing polling interval, all to no avail.

**Comment by Forbo** [comment]
Forgot to mention I'm running version 1.2.5 from F-Droid.

**Comment by Forbo** [comment]
Turns out it eventually worked for one of the reminders. I had the polling interval set to 5 seconds, and after about an hour of actively using the phone a reminder finally triggered.

**Comment by ildar** [comment]
Did you try the different mode of operation? (poll interval == 0)

**Comment by Forbo** [comment]
Yes, that was the initial state when I set up the reminders. After it had failed to issue the reminder upon connecting to the network, that's when I began experimenting with other intervals.
